backup.config-example

```diff
@@ -43,3 +43,9 @@ GHE_NUM_SNAPSHOTS=10
 #
 # WARNING: do not enable this, only useful for debugging/development
 #GHE_BACKUP_FSCK=no
+
+# In some operatingsystems various tools such as gnu are located elsewhere
+# Modify you path here.
+#
+# Set this path on Solaris 11
+#PATH="/usr/gnu/bin:$PATH:/usr/sbin"
```

Comment:
As we document in the README.md, we rely on the comments in the configuration file to explain the purpose of each settings so I think this is a good place and generic enough for most sysadmins to find and understand, regardless of operating system.  I think it's only a matter of getting the wording correct to keep things generic whilst at the same time conveying that this is what needs to be modified, regardless of OS, if commands are not in the path of the executing user.

> I worked for Sun+Oracle 10+ years.

So did I. 👋 fellow Sun alumnus.